.codecov.yaml

```diff
@@ -0,0 +1,4 @@
+ignore:
+  - "**/zz_generated*.go" # generated
+  - "dapr/proto"
+  - "example" 
```

Comment:
is it fine to use just directory ?
```
"dapr/proto/**"
"example/**"
```

Reply:
Looks like it is allowed https://docs.codecov.io/docs/ignoring-paths . I wasn't aware in the other PR.